src/Utils/SchemaExtender.php

```diff
@@ -56,6 +56,139 @@ class SchemaExtender
 
     protected static ASTDefinitionBuilder $astBuilder;
 
+    /**
+     * @param array<string, bool> $options
+     * @phpstan-param TypeConfigDecorator|null $typeConfigDecorator
+     *
+     * @api
+     */
+    public static function extend(
```

Comment:
Is it necessary to move things around like this? It makes it hard to see what has actually changed…